packages/core/types/index.js

```diff
@@ -1754,6 +1755,7 @@ export type PackagingProgressEvent = {|
   +type: 'buildProgress',
   +phase: 'packaging',
   +bundle: NamedBundle,
+  +bundleGraph?: InternalBundleGraph,
```

Comment:
1. Do we really want to add the bundle graph to this event? (This is the public plugin API)
2. If yes, then it certainly needs to be the public bundle graph type

Reply:
Hmm I think the main reason why I added this was because the ReporterRunner gets the BundleGraph from the WorkerFarm, which wouldn't exist in this case since we don't serialize the BundleGraph, so I sent the BundleGraph through the packaging/optimizing events.

https://github.com/parcel-bundler/parcel/blob/cf5b3ffdaadb3de90104a6d24261e1aad9c496fd/packages/core/core/src/ReporterRunner.js#L67-L73

The invariant here also looks like it uses the private BundleGraph type?

Reply:
Maybe we should make an internal ReporterEvent type that has this instead?